app/controllers/v0/sign_in_controller.rb

```diff
@@ -129,10 +130,11 @@ def refresh
       sign_in_logger.info('refresh error', { errors: e.message })
       StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_REFRESH_FAILURE)
       render json: { errors: e }, status: :bad_request
-    rescue SignIn::Errors::StandardError => e
+    rescue SignIn::Errors::StandardError, Login::Errors::CredentialLockedError => e
+      status = e.is_a?(Login::Errors::CredentialLockedError) ? :forbidden : :unauthorized
```

Comment:
I've seen some arguments that `unauthorized` could be the appropriate thing to use in this case. I'm not sure I agree, however, it is simpler, so I like the idea.  I recommend just returning the `unauthorized` status regardless here